brave/brave-core
# Add ephemeral storage support for localStorage and sessionStorage

Resolves https://github.com/brave/brave-browser/issues/11619

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

 * Open browser with clean profile, pointed to rewards prod.
 * Enable third-party cookies
 * Enable "Ephemeral Storage" (#brave-ephemeral-storage) in `about:flags`
 * Navigate to https://dev-pages.brave.software/storage/frames.html
 * Ensure that the test on that page works properly.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
